### ISSUE TITLE
Added DiffAI

### ISSUE THREAD

**Issue by mmirman** [robust-ml/robust-ml.github.io#1]
**Name**: Differentiable Abstract Interpretation for Provably Robust Neural Networks

**Authors**: Matthew Mirman, Timon Gehr, Martin Vechev 

**Paper**: https://www.sri.inf.ethz.ch/papers/icml18-diffai.pdf

**Code**: https://github.com/eth-sri/diffai

**Venue**: ICML 2018

**Does the code implement the robust-ml API and include pre-trained models**: no

**Dataset**: MNIST

**Threat model**: $l_\infty(\epsilon = 0.1)$

**Natural accuracy**: 99%

**Claims**: 96.4% proved

**Comment by anishathalye** [comment]
Thank you for submitting this -- we'd love to include it in the listing.

We require that both defenses and attacks implement the [robustml](https://github.com/robust-ml/robustml) interface. Would you be able to add this? It usually takes only several minutes and tens of lines of code (see an example [here](https://github.com/MadryLab/cifar10_challenge/blob/master/model_robustml.py)).

**Comment by mmirman** [comment]
Sure, we'll do that

**Comment by anishathalye** [comment]
Awesome! Ping us when you're done, and we'll add the defense to the list.

**Comment by anishathalye** [comment]
Bump -- do you have any questions or issues with implementing the interface?

**Comment by mmirman** [comment]
On pause until we update our repo with some larger changes

**Comment by anishathalye** [comment]
Bump - any update on this?
